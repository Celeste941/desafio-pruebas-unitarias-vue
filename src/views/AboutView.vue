<template>
  <div class="about">
    <h1>SecciÃ³n about</h1>
  </div>
</template>
