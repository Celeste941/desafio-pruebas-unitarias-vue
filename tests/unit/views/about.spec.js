import { mount } from "@vue/test-utils";
import AboutView from "@/views/AboutView.vue";

describe("Test de vista aboutView.vue", () => {
  it("Match de snapshot con AboutView.vue", () => {
    const wrapper = mount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});