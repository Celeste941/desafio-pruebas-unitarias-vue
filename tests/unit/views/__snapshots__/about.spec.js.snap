// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de vista aboutView.vue Match de snapshot con AboutView.vue 1`] = `
<div class="about">
  <h1>Secci√≥n about</h1>
</div>
`;
